### ISSUE TITLE
[sklearn#34366] Accessibility issues in documentation website

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#332]
**Mirrored from [scikit-learn#34366](https://github.com/scikit-learn/scikit-learn/issues/34366)**

Original title: Accessibility issues in documentation website
Original labels: Documentation, frontend
Created: 2026-06-22T16:31:27Z

---

According to "WAVE" (a web accessibility evaluation tool) there are several issues on our website. 
So far, I have tested only two pages.

The home page:

<img width="1634" height="1001" alt="Image" src="https://github.com/user-attachments/assets/928253ef-5bfd-4a2e-84d8-1b8529868712" />

The report can be seen here https://wave.webaim.org/report#/https://scikit-learn.org/stable

<br><br>
An example plot:
<img width="1441" height="990" alt="Image" src="https://github.com/user-attachments/assets/af5d0edc-4526-4264-aec3-f15b5ba80f10" />

Report: https://wave.webaim.org/report#/https://scikit-learn.org/stable/auto_examples/compose/plot_column_transformer_mixed_types.html

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Addressed in upstream PR(s). See scikit-learn/scikit-learn PRs for #34366.
